legal-api/tests/unit/models/test_business.py

```diff
@@ -254,6 +261,35 @@ def test_good_standing(session, last_ar_date, legal_type, state, limited_restora
 
     assert business.good_standing is expected
 
+RESTORATION_FILING = copy.deepcopy(FILING_HEADER)
+RESTORATION_FILING['filing']['restoration'] = RESTORATION
```

Comment:
Can you add 2 blank lines between constant values and the tests? It will make it more readable.

Reply:
addressed, a good practice learned ✔️